README.md

```diff
@@ -1,16 +1,40 @@
-# Release Integration Tests
+# Release Integration Test
 
+This README provides instructions on running a test using the provided shell script.
```

Comment:
I am confused. Is this a README for the repo or a README for just the fbc test? Let's make it clear

Reply:
This README for now is for the only test in repo , for the fbc test